src/FlipMove.js

```diff
@@ -69,7 +69,7 @@ class FlipMove extends Component {
     // can complete. Because we cannot mutate props, we make `state` the source
     // of truth.
     this.state = {
-      children: props.children.map(child => ({
+      children: Children.toArray(props.children).map(child => ({
```

Comment:
Ahh I see, so the issue is that React-like libraries may mutate the supplied props in-between layers. Even though we supply an array as the `children` prop, libraries internally munge it into a format that works for them.

Thanks for the clarification.